syncer/syncer.go

```diff
@@ -2430,6 +2436,7 @@ func (s *Syncer) setSyncCfg() {
 		UseDecimal:              true,
 		VerifyChecksum:          true,
 		TimestampStringLocation: s.timezone,
+		MaxReconnectAttempts:    maxBinlogSyncerReconnect,
```

Comment:
How about giving more config items like `ReadTimeout`, `HeartbeatPeriod`, etc. (also see what we did in relay, https://github.com/pingcap/dm/blob/1724a39f692145faaec7e4f878b1ec622bbd0a74/relay/relay.go#L120).

Reply:
update in f2ca52f